spec/system/page_objects/components/select_kit.rb

```diff
@@ -26,8 +26,7 @@ def hidden?
       end
 
       def expanded_component
-        return expand if is_collapsed?
-        find(@context + ".is-expanded", wait: 5)
```

Comment:
Note we shouldn't be hard coding `wait` here as it overwrites `Capybara.default_wait_time`